assignments/tasks.go

```diff
@@ -0,0 +1,284 @@
+package assignments
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"path/filepath"
+
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/scm"
+)
+
+// Things to do:
```

Comment:
Please use the marker `TODO(name): <task to do>` since this allows easier to search for TODO issues in code.